Sources/Filters+Strings.swift

```diff
@@ -60,8 +88,21 @@ extension Filters {
       let transformed = String(scalars[start..<idx]).lowercased() + String(scalars[idx..<scalars.endIndex])
       return transformed
     }
-
+    
+    /// Deprecated in favor of `upperFirstLetter`
     static func titlecase(_ value: Any?) throws -> Any? {
```

Comment:
Just remove this function, and point the `registerFilter` call to the function below.